GameServer/Logic/Game.cs

```diff
@@ -36,22 +37,57 @@ public class Game
         // Object managers
         private ItemManager _itemManager;
         // Other managers
-        private ChatboxManager _chatboxManager;
+        private ChatCommandManager _chatCommandManager;
         private PlayerManager _playerManager;
         private NetworkIdManager _networkIdManager;
 
         public Game(
             ItemManager itemManager,
-            ChatboxManager chatboxManager,
+            ChatCommandManager chatCommandManager,
             NetworkIdManager networkIdManager,
             PlayerManager playerManager,
             Logger logger
         ) {
             _itemManager = itemManager;
-            _chatboxManager = chatboxManager;
+            _chatCommandManager = chatCommandManager;
             _networkIdManager = networkIdManager;
             _playerManager = playerManager;
             _logger = logger;
+            InitializeCommands();
+        }
+
+        private void InitializeCommands()
```

Comment:
Just move this to the `ChatCommandManager` and call it `LoadCommands` (commands will be loaded from LUA files as well at some point, so loading would make more sense than Initializer) and this is good to go :+1: